.editorconfig

```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
I'll agree adding editor conventiosn like this is good. But it's probably better to be in its own PR
to avoid unrelated changes about adding support for loongarch.